http_service/pkg/http_user_service/repository.go

```diff
@@ -0,0 +1,38 @@
+package httpuserservice
+
+import (
+	"context"
+
+	"github.com/go-kit/log"
+	"github.com/go-kit/log/level"
+	"github.com/jcromanu/final_project/http_service/errors"
+	"github.com/jcromanu/final_project/http_service/pkg/entities"
+	"github.com/jcromanu/final_project/user_service/pb"
+	"google.golang.org/grpc"
+)
+
+type HttpRepository interface {
```

Comment:
this interface  must be in the service file, o wherever it is being used.

Reply:
aditional can you name the interface just as repository, it don cares if the repo is  http, grpc, mongo etc.

Reply:
ok ,I change it to a generic Repository name